src/components/SelectedTapMobile/SelectedTapMobileInfo.module.scss

```diff
@@ -1,12 +1,13 @@
-$dark-grey: #2D3748;
-$light-grey: #60718C;
-$lighter-grey: #E9EEF4;
+$dark-grey: #2d3748;
+$light-grey: #60718c;
+$lighter-grey: #e9eef4;
 
 .halfInfo {
   & * {
     margin: 0;
     font-size: 14px;
-    }
+    height: 100vh;
+  }
```

Comment:
The only actual code change: this worked on my browser when I set the view to responsive.